### DISCUSSION TITLE
Visual expression

### DISCUSSION THREAD

**Discussion by daniel-vnext** [alan-turing-institute/AutSPACEs#373] [Ideas]
# Overview

*As a person affected by autism
I want to be able to express myself visually
So that I can better communicate my experience*

## Idea

People affected by autism sometimes find it much easier to express themselves through visual and graphical art forms. The idea is to provide a facility to do this, rather than through text or voice

This is not related to the non verbal pictorial editor

**Comment by GeorgiaHCA** [comment]
This user story is based on recommendations from an Autistic commnuity member given at an open call to showcase work on the project. 

The original notes fom that meeting which this user story is taken and adapted from are here: https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/community-recommendations/workshops/developer-showcase-August2020.md

**Comment by GeorgiaHCA** [comment]
This would be an amazing thing to be able to do, however it is beyond the current scope of this project since the data would be incompatible with the other datat and hard to read.

An alternative option would be to allow users to upload images in addition to explanatory text. It allows for greater expression, but does not deal with the problem of accessibility.

**Comment by lottycoupat** [comment]
@GeorgiaHCA could this be added as a "requirement" but as a "would have" category?

**Comment by GeorgiaHCA** [comment]
I think that this is not yet on the requirements list. It's more of a "backlog" item since it is beyond the current scope (and the research implications would need working out)

**Comment by GeorgiaHCA** [comment]
But it's useful to have on the repo - in case someone in the open source community in the future is able to respond to this or do work on it
